src/hooks/useObject.ts

```diff
@@ -0,0 +1,128 @@
+import { FormContextValues } from 'react-hook-form'
+
+import {
+  UseObjectProperties,
+  UseObjectReturnType,
+  BasicInputReturnType,
+  UISchemaType,
+  UITypes,
+} from './types'
+import {
+  JSONSchemaType,
+  useObjectFromPath,
+  concatFormPath,
+} from '../JSONSchema'
+import { getGenericInput } from './useGenericInput'
+import { getInputCustomFields } from './useInput'
+import { getRadioCustomFields } from './useRadio'
+import { useFormContext } from '../components'
+import { getSelectCustomFields } from './useSelect'
+import { getHiddenCustomFields } from './useHidden'
+import { getPasswordCustomFields } from './usePassword'
+import { getTextAreaCustomFields } from './useTextArea'
+
+function getFromGeneric(
+  genericInput: BasicInputReturnType
+): UseObjectReturnType {
+  const currentObject = genericInput.getObject()
+  const inputs: UseObjectReturnType = []
+
+  switch (currentObject.type) {
```

Comment:
Should something happen in the default of switch case?

Reply:
No